designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/DeploymentManager.scala

```diff
@@ -139,6 +139,13 @@ case object NoDeploymentSynchronisationSupport extends DeploymentSynchronisation
 
 sealed trait SchedulingSupport
 
+trait BaseSchedulingSupported extends SchedulingSupported {
```

Comment:
do we need the trait? maybe we could add the defaults to `SchedulingSupported`?

Reply:
I agree, defaults added